src/components/custom/useCustomDialog.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+
+const defaultValues = {
+  d_title: 'Ooops',
```

Comment:
In JS world lowerCamelCase is de facto the standard. Please rename props accordingly.